### PR TITLE
Adds map-filter auto zoom behaviour setting

### PR DESCRIPTION
**Pull request by aj-he**
### Types of changes

-   [ ] Bugfix (non-breaking change which fixes an issue)
-   [x] New feature (non-breaking change which adds functionality)
-   [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### Description of Change

Following conversation in previous PR #10195, removed UX components in favour of a larger system wide configuration UX.

The basic logic remains, where a setting determines the map-filter's behaviour to automatically moves the map extent to encompass the current search results aggregate bounds. 

### Issues Solved

#9564 

### Checklist

-   [x] Unit tests pass locally with my changes
-   [ ] I have added tests that prove my fix is effective or that my feature works
-   [ ] I have added necessary documentation (if appropriate)

#### Ticket Background
*   Sponsored by: Historic England
*   Found by: @aj-he 
*   Tested by: @aj-he  
*   Designed by: @aj-he

### CODE REVIEW

File: arches/app/templates/javascript.htm [hunk 1]
```diff
@@ -959,6 +959,7 @@
             "label": "{{ concept_collection.label | escapejs }}",
             "id": "{{ concept_collection.id }}"
         },{% endfor %}]'
+        cookie-prefix="{{ app_settings.SESSION_COOKIE_NAME }}"
```
Comment (chrabyrd) [inline arches/app/templates/javascript.htm:962]: Could you remove this line since we're no longer using cookies for this interaction?

**Review by chrabyrd — CHANGES_REQUESTED** [review]
Overall looks good!

Just one minor request and it should be good to go 👍

File: arches/app/templates/javascript.htm (hunk 1 above)
Comment (aj-he) [reply arches/app/templates/javascript.htm]: @chrabyrd - good catch. I've removed the offending line.

**Review by chrabyrd — APPROVED** [review]
lgtm!

**Pull request merged**